x-pack/plugins/cases/public/components/status/button.tsx

```diff
@@ -13,20 +13,14 @@ import { statuses } from './config';
 
 interface Props {
   status: CaseStatuses;
-  disabled: boolean;
   isLoading: boolean;
   onStatusChanged: (status: CaseStatuses) => void;
 }
 
 // Rotate over the statuses. open -> in-progress -> closes -> open...
 const getNextItem = (item: number) => (item + 1) % caseStatuses.length;
 
-const StatusActionButtonComponent: React.FC<Props> = ({
-  status,
-  onStatusChanged,
-  disabled,
-  isLoading,
-}) => {
+const StatusActionButtonComponent: React.FC<Props> = ({ status, onStatusChanged, isLoading }) => {
```

Comment:
The status button will always look enabled. We'll remove the arrow and `onClick` if the user is readonly though. That way we can keep the color of the status.